Dockerfile

```diff
@@ -1,8 +1,11 @@
-FROM yunspace/alpine-golang
+FROM golang:alpine
 
-ADD build/codeclimate-gofmt /usr/src/app/
+WORKDIR /usr/src/app
```

Comment:
FYI: having `/code` be the WORKDIR is a requirement of the SPEC. Maybe you should double check that this Dockerfile is in accordance in general?